src/main/java/pl/cinema/server/customer/dto/CustomerDto.java

```diff
@@ -0,0 +1,16 @@
+package pl.cinema.server.customer.dto;
+
+import lombok.Builder;
+import lombok.Value;
+
+
+@Value
+@Builder
+public class CustomerDto {
+    private Long id;
+    private String name;
+    private String surname;
+    private Long age;
```

Comment:
Długowiecznych planujesz gości :) Po co wiek? Zwłaszcza że taka zmienna się deaktualizuje, nie śledzi wieku.

Reply:
tak, można użyć innego typu.
Hmm to racja, dobra uwaga, czas leci, a w bazie nadal ten sam wiek... wystarczy więc w dto MakerDto, przy rezerwacji poda wiek klient i później nie musimy tego pamiętać